client/src/components/menubar/infoMenu.js

```diff
@@ -1,31 +1,37 @@
 // jshint esversion: 6
 import React from "react";
 import { Button, Popover, Menu, MenuItem, Position } from "@blueprintjs/core";
-import { IconNames } from "@blueprintjs/icons";
 import styles from "./menubar.css";
 
 const handleClick = (dispatch) => {
   dispatch({ type: "toggle dataset drawer" });
 };
 
 const InformationMenu = React.memo((props) => {
-  const { libraryVersions, tosURL, privacyURL, dispatch } = props;
+  const {
+    libraryVersions,
+    tosURL,
+    privacyURL,
+    auth,
+    userinfo,
+    dispatch,
+  } = props;
   return (
     <div className={`bp3-button-group ${styles.menubarButton}`}>
       <Popover
         content={
           <Menu>
             <MenuItem
               onClick={() => handleClick(dispatch)}
-              icon={IconNames.BOOK}
+              icon="info-sign"
               text="Dataset Overview"
             />
 
             <MenuItem
               href="https://chanzuckerberg.github.io/cellxgene/"
               target="_blank"
```

Comment:
sorry, this is existing code, but we should be adding rel="noopener" to any new tab links for security.

https://web.dev/external-anchors-use-rel-noopener/